components/icon/stories/icon.test.js

```diff
@@ -1,99 +1,127 @@
-                            "row-gap": "48px",
-                            "place-items": "center",
-                        })}
-                    >
-                        ${["xs","s","m","l","xl","xxl"].map(
-                            (size) => Template({ ...args, ...row_args, size })
-                        )}
-                    </div>`
-                )}
-            </div>
-            <div
-                style=${styleMap({
-                    "display": "flex",
-                    "flex-direction": "column",
-                    "gap": "16px",
-                })}
-            >
-                ${uiIconsWithDirections.map(iconName => html`
-                    <div
-                        style=${styleMap({
-                            "display": "grid",
-                            "grid-template-columns": "repeat(8, 1fr)",
-                            "column-gap": "24px",
-                            "row-gap": "48px",
-                        })}
-                    >
-                        ${uiIconSizes[iconName.replace(/(Left|Right|Up|Down)$/, "")]?.map((iconSize) =>
-                            Template({ ...args, setName: "ui", iconName: iconName + iconSize }, context)
-                        )}
-                        ${when(uiIconSizes[iconName]?.length == 0, () =>
-                            Template({ ...args, setName: "ui", iconName }, context)
-                        )}
-                    </div>`
-                )}
-            </div>
-        </div>
-    `;
-};
+export const TestTemplate = (args, context) => html`
+	<div style=${styleMap({
+		"display": "flex",
+		"gap": "12px",
+	})}>
+		<div>
+			${Typography({
+				semantics: "detail",
+				size: "l",
+				content: ["Workflow icons"],
+				customStyles: {
+					"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+				}
+			})}
+			<div
+				style=${styleMap({
+				"display": "grid",
+				"grid-template-columns": `repeat(${workflowSizes.length}, 50px)`,
+				"gap": "16px",
+				"border": "1px solid var(--spectrum-gray-200)",
+				"border-radius": "4px",
+				"padding": "16px",
+				"margin-block-end": "32px",
+			})}
+			>
+				${workflowSizes.map(scale => html`
+					<div>
+						${Typography({
+							semantics: "detail",
+							size: "s",
+							content: [scale],
+							customStyles: {
+								"--mod-detail-font-color": "var(--spectrum-seafoam-900)",
+							}
+						})}
+					</div>
+				`)}
+				${[
+					"Add",
+					"AlertTriangle",
+					"Edit",
+					"File",
+					"Folder",
+					"MenuHamburger",
+					"SelectRectangle",
+					"Redo",
+					"Star",
+					"StarFilled",
+					"User",
+				].map((iconName, idx) => html`
+					${workflowSizes.map((size) => Template({
+						...args,
+						useRef: true,
+						iconName,
+						setName: "workflow",
+						size,
+						fill: idx % 5 === 0 ? "var(--spectrum-negative-content-color-default)" : undefined
+					}, context))}
+				`)}
+			</div>
+		</div>
+		<div>
+			${Typography({
```

Comment:
Do these typography elements need to leverage the chromatic-ignore class? Or are these not used in the snapshots?

Reply:
I've added the `chromatic-ignore` class to these 👍